sass/main.scss

```diff
@@ -754,10 +815,306 @@ footer {
+            justify-content: center;
+
+            .tape {
+                margin-top: -10px; //Half of tape height
+                height: 20px;
+            }
+        }
+
+        #schedule-content {
+            flex-direction: column;
+            padding: 20px;
+
+            #days {
+                width: auto;
+                margin-top: 10px;
+                flex-direction: row;
+
+                .day {
+                    .number {
+                        font-size: 5em;
+                        text-align: center;
+                    }
+                    .name {
+                        font-size: 1.5em;
+                    }
+                }
+            }
+
+            .events {
+                width: auto;
+
+                .event {
+                    padding: 5px;
+                    .title {
+                        font-size: 1em;
+                    }
+                }
+            }
+        }
+    }
+
+    #faq #faq-wrapper {
+
+        h1 {
+            margin-bottom: 20px;
+        }
+
+        #faq-cards {
+            flex-direction: column;
+
+            &-left, &-right {
+                margin: 0;
+                width: 100%;
+            }
+
+            .card {
+                .accordion-header {
+                    padding: 10px 20px;
+                    font-size: 1em;
+                }
+                .panel {
+                    padding: 20px;
+                    margin-left: 30px;
+                     // TODO: Why does padding: 10px not look good?
+                    padding-bottom: 10px;
+
+                    p {
+                        margin-bottom: 0;
+                    }
+                }
+            }
+        }
+    }
+
+    #leadership #leaders {
+        grid-template-columns: repeat(3, $leadership-size-mobile);
+        justify-content: space-around;
+        gap: 20px;
+    }
+}
+
+@media screen and (max-width: 650px) {
+
+    .section-pad {
+        padding: 20px;
+    }
+
+    #hero {
+
+        p {
+            line-height: 0.4;
+        }
+
+        #bh7 {
+            font-size: 2.4em;
+        }
+
+        #premiere {
+            font-size: 1.5em;
+        }
+
+        #dates {
+            font-size: 0.7em;
+        }
+
+        #desk-container {
+            right: -160px;
+            width: 160%;
+        }
+
+        #register {
+           @include button;
+            padding: 10px 40px;
+            font-size: 1.2em;
+        }
+
+    }
+
+    #hackathon {
+        .carousel .slick-arrow {
+            display: none !important; // Overrides slick.css
+        }
+
+        #hackathon-content #mlh {
+            margin: 0;
+            padding: 20px;
+
+            i {
+                font-size: 1.5em;
+            }
+            p {
+                font-size: 1em;
+            }
+        }
+    }
+
+    #faq #faq-wrapper {
+        // Only center the
+        a {
+            white-space: nowrap;
+        }
+        #contact  {
```

Comment:
again: new line between ending bracket and new selector?

Reply:
I'm seeing more of this above were I started noticing it

please keep your formatting consistent